communications/meetings.md

```diff
@@ -7,58 +7,49 @@ Use the guides below as a starting point, and adapt for specific situations acco
-- Edit meeting agenda to reflect the outcomes of the discussion meeting note.
-- Put a tentative timebox on each agenda item.
-
-Tips:
-- View "All" meetings on Hugo to fit the practice of having pre-meeting notes for discussions.
-
-![all meetings](https://user-images.githubusercontent.com/2837532/76875868-8580d100-6847-11ea-95c8-532b29ea6d6d.png)
-
-- "Import Agenda" can be used as a quick starting point for any meeting note, especially the one used for agenda discussions.
-
-<img width="290" alt="import agenda" src="https://user-images.githubusercontent.com/2837532/76816543-e58b5f00-67d6-11ea-893a-a411a4a24bf8.png">
-
-- Hugo syncs notes discussions as Slack threads (two-way). Participants can freely discuss on either platform.
-
-<img width="562" alt="sync" src="https://user-images.githubusercontent.com/2837532/76794286-0d5dd100-679d-11ea-80b9-2077e2c6ce3f.png">
+Guide for Organizers:
+- Doublecheck the intended attendees to the meeting and ensure they have all received calendar invitations.
+    - *Tip*: [How to use Zoom for meetings?](https://docs.google.com/document/d/1WNqeFKXYv_ji_PUXBKq8nlD09ErqmB6VoWasPW-BUDI/edit?usp=sharing)
+- Initiate a **Meeting Note** on Hugo, provide as much context as possible. Sync it to a relevant Slack channel to prompt for pre-meeting discussions. ![notes-hugo](https://user-images.githubusercontent.com/2837532/104062313-3a65ef00-51c8-11eb-9a0c-005134c16473.png) ![notes-slack-thread](https://user-images.githubusercontent.com/2837532/103943174-9d3f8380-50ff-11eb-9377-b839d780e10a.png)
+    - Make sure links to all relevant external references (such as Google Docs, Zoom recordings, Zeplin designs, Notion notes, etc.) are comprehensively included in the Hugo notes and make sure that all the participants have access to those.
+    - *Tip*: Use Hugo's "New Note" instead of "Prepare" for meeting notes that are shareable to Slack channels that allow synchronized comments and thread replies.
+    - *Tip*: Use Hugo's "Import Agenda" feature as a quick starting point for any meeting note, especially the one used for agenda discussions.
+
+Guide for Participants:
+- Go through relevant context, examples (where applicable):
+    - Meeting notes and their associated pre-meeting discussions prepared by the organizer
+    - Summary of pre-meeting discussions through the use of the `/notes` command. ![notes-discussion-summary](https://user-images.githubusercontent.com/2837532/103942564-9bc18b80-50fe-11eb-9c3b-12dde574c43e.png)
+    - Weekly digest (and daily updates) via the [automated updates feed](https://github.com/eqworks/updates). ![updates](https://user-images.githubusercontent.com/2837532/103942697-cb709380-50fe-11eb-8951-116c126e229c.png)
+- Raise questions, proposals, and help resolve issues under the pre-meeting discussion thread. ![discussion](https://user-images.githubusercontent.com/2837532/103943386-feffed80-50ff-11eb-9fdf-16c5cbcfb3f5.png)
```

Comment:
so in case of not every attendees read this, should proposals and questions be also raised during the meeting, maybe in a shortened form?

Reply:
sure. the _recommendation_ is to have as much questions/proposals be raised before the meeting so the anticipation of the meeting is pre-determined. _but_ the case you mentioned and improvised/reactive questions sure should be raised during meetings, just need to be a bit more controlled mostly through timeboxing and collective moderation.

i'll add this as a part of the "During A Meeting" section